core/test/integration/data/importer/importers/data_spec.js

```diff
@@ -1259,8 +1259,8 @@ describe('Import (new test structure)', function () {
                 users = importedData[0];
                 rolesUsers = importedData[1];
 
-                // we imported 3 users, there were already 3 users, only one of the imported users is new
-                users.length.should.equal(6, 'There should only be 6 users');
+                // we imported 4 users, there were already 3 users, only one of the imported users is new
```

Comment:
the desc seems wrong.

> we imported 4 users

we still import 3, but 4 existing users, because you have extended the test utilities.